src/index.html

```diff
@@ -19,9 +19,367 @@
       rel="stylesheet"
       href="styles/index.scss"
     />
+    <link
+      rel="stylesheet"
+      href="styles/reset.scss"
```

Comment:
The link tag is referencing a SCSS file (`styles/reset.scss`). Ensure that this SCSS file is compiled to CSS before being linked in the HTML, as browsers do not understand SCSS directly.